routes/movies.routes.js

```diff
@@ -0,0 +1,68 @@
+const Celebrity = require("../models/Celebrity.model");
+const Movie = require("../models/Movie.model");
+
+// starter code in both routes/celebrities.routes.js and routes/movies.routes.js
+const router = require("express").Router();
+
+// all your routes here
+
+router.get("/create", (req, res) => {
+    Celebrity.find()
+    .then(allCelebs => res.render("movies/new-movie", allCelebs))
+    .catch(err => console.log(err))
+    // res.render("movies/new-movie");
+})
+
+router.post("/create", (req, res) => {
+    const { title, genre, plot,cast } = req.body;
+  
+    Movie.create({ title, genre, plot, cast })
+      .then(celeb => res.render("index", celeb))
+      .catch(err => console.log(err))
+})
+
+router.get("/movies", (req, res) => {
+
+    Movie.find()
+      .then(allMovies => res.render("movies/movies", { allMovies }))
+      .catch(err => console.log(err))
+});
+
+router.get("/:id", (req, res) => {
+    const id = req.params.id
+  
+    Movie.findById(id)
+      .populate("celebrity")
+      .then(movie => {
+        res.render("movies/movie-details", movie)
+      })
+      .catch(err => console.log(err))
+});
+
+router.post("/:id/delete", (req, res) => {
+    const id = req.params.id
+    Movie.findByIdAndDelete(id)
+    .then(console.log(id))
+      .then(res.render("index"))
+      .catch(err => console.log(err))
+
+});
+
+router.get("/:id/edit", (req, res) =>{
+    const id = req.params.id
+
+    Movie.findById(id)
+        .then(movie => res.render("movies/edit-movie", movie))
+        .catch(err => console.log(err))
+})
```

Comment:
- Aquí te hace falta popular, para así incluir el objeto entero de `Celebrity` en vez del id que hay dentro del array de la propiedad `cast`, y así en la vista puedes llamar a `name` y que se muestre en el formulario de edición.

```suggestion
router.get("/:id/edit", (req, res) =>{
    const id = req.params.id

    Movie.findById(id)
        .populate("cast")
        .then(movie => res.render("movies/edit-movie", movie))
        .catch(err => console.log(err))
})
```